docs/ECLLanguageReference/ECLR_mods/Recrd-ScopeNLogicalFilenames.xml

```diff
@@ -13,16 +13,26 @@
         <primary>File Scope</primary>
       </indexterm></title>
 
-    <para><emphasis role="bold">The logical filenames used in DATASET and
-    INDEX attribute definitions and the OUTPUT and BUILD (or BUILDINDEX)
-    actions may begin with a scope name, indicated by a leading tilde (~), and
-    may contain directories. The scope and directories are delimited by double
-    colons (::). The presence of a scope in the filename allows you to
-    override the default scope name for the cluster.</emphasis></para>
-
-    <para>For example, assuming you are operating on a cluster whose default
-    scope name is “Training” then the following two OUTPUT actions result in
-    the same scope:</para>
+    <para>The logical filenames used in
+    DATASET and INDEX attribute definitions and the OUTPUT and BUILD (or
+    BUILDINDEX) actions can optionally begin with a ~ meaning it is absolute,
+    otherwise it is relative (the platform configured scope prefix is
+    prepended). It may contain directories. The scope and directories are
```

Comment:
Not sure appropriate to mention directories here.
Scopes happen to map to directories in our current implementation, but in theory could not. I think it's better just to talk about scopes here.

The exception is in a physical file specification (i.e. ~file::...), but you have a dedicated section for those under the heading of "Landing_Zone_Files". 
Similarly you don't mention ~foreign syntax in this section, as you have a dedicated section for it.